mappings/net/minecraft/block/RedstoneWireBlock.mapping

```diff
@@ -25,34 +32,46 @@ CLASS net/minecraft/class_2457 net/minecraft/block/RedstoneWireBlock
 		ARG 1 state
 	METHOD method_10487 getWireColor (I)I
 		ARG 0 powerLevel
-	METHOD method_27840 (Lnet/minecraft/class_1922;Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;
+	METHOD method_27840 getBlockView (Lnet/minecraft/class_1922;Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;
 		ARG 1 world
 		ARG 2 state
 		ARG 3 pos
+	METHOD method_27841 getSideConnectionState (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;Z)Lnet/minecraft/class_2773;
+		ARG 1 world
+		ARG 2 position
+		ARG 3 direction
+		ARG 4 isTransparentAbove
 	METHOD method_27842 getReceivedRedstonePower (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)I
 		ARG 1 world
 		ARG 2 pos
-	METHOD method_27843 (Lnet/minecraft/class_1922;Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;
+	METHOD method_27843 getConnectionState (Lnet/minecraft/class_1922;Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;
 		ARG 1 world
 		ARG 2 state
 		ARG 3 pos
-	METHOD method_27844 (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)V
+	METHOD method_27844 provideNeighborUpdates (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)V
 		ARG 1 world
 		ARG 2 pos
-	METHOD method_27845 (Lnet/minecraft/class_2680;)Lnet/minecraft/class_265;
+	METHOD method_27845 makeOutlineShape (Lnet/minecraft/class_2680;)Lnet/minecraft/class_265;
 		ARG 1 state
 	METHOD method_27846 isFullyConnected (Lnet/minecraft/class_2680;)Z
 		ARG 0 state
-	METHOD method_27936 (Lnet/minecraft/class_1937;Ljava/util/Random;Lnet/minecraft/class_2338;Lnet/minecraft/class_1160;Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;FF)V
+	METHOD method_27936 emitDustParticle (Lnet/minecraft/class_1937;Ljava/util/Random;Lnet/minecraft/class_2338;Lnet/minecraft/class_1160;Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;FF)V
 		ARG 1 world
 		ARG 2 random
 		ARG 3 pos
+		ARG 4 particleColor
```

Comment:
A Vector3f can't be a particle color

Reply:
Hmm, I think mojang usese 3 floats for color in some places